src/components/tab-caption/tab-caption.tsx

```diff
@@ -124,6 +125,6 @@ export class TabCaption implements GxComponent, DisableableComponent {
 
   private clickHandler(event: UIEvent) {
     event.preventDefault();
-    this.selected = true;
+    this.selected = !this.disabled;
```

Comment:
Could a tab caption be initially disabled and selected? If yes, what would happen if the tab caption is clicked?

Reply:
Visually it is fine, but it is a potential bug.